about.md

```diff
@@ -6,59 +6,31 @@ title: About
-The Earth System Model Evaluation Tool (ESMValTool) is a community
-diagnostics and performance metrics tool for the evaluation of Earth
-System Models (ESMs) that allows for routine comparison of single or
-multiple models, either against predecessor versions or against
-observations. The priority of the effort so far has been to target
-specific scientific themes focusing on selected Essential Climate
-Variables, a range of known systematic biases common to ESMs, such as
-coupled tropical climate variability, monsoons, Southern Ocean
-processes, continental dry biases and soil hydrology-climate
-interactions, as well as atmospheric CO2 budgets, tropospheric and
-stratospheric ozone, and tropospheric aerosols. The tool is being
-developed in such a way that additional analyses can easily be added.
-A set of standard recipes for each scientific topic reproduces specific
-sets of diagnostics or performance metrics that have demonstrated their
-importance in ESM evaluation in the peer-reviewed literature. The
-ESMValTool is a community effort open to both users and developers
-encouraging open exchange of diagnostic source code and evaluation
-results from the CMIP ensemble. This will facilitate and improve ESM
-evaluation beyond the state-of-the-art and aims at supporting such
-activities within the Coupled Model Intercomparison Project (CMIP) and
-at individual modeling centers. Ultimately, we envisage running the
-ESMValTool alongside the Earth System Grid Federation (ESGF) as part of
-a more routine evaluation of CMIP model simulations while utilizing
-observations available in standard formats (obs4MIPs) or provided by
-the user.
+
+The Earth System Model Evaluation Tool (ESMValTool) is an open-source community-developed diagnostics
+and performance metrics tool for the evaluation and analysis of Earth System Models (ESMs). ESMValTool
+allows for a comparison of single or multiple models, either against predecessor versions or against
+observations. The aim of the ESMValTool is to take model evaluation to the next level by
+facilitating analysis of many different ESM components, providing well-documented source code and
+scientific background of implemented diagnostics. Traceability and reproducibility of the results are
+ensured by providing detailed provenance records for all output. While originally designed to facilitate
+a more comprehensive and rapid evaluation of models participating in the
+[Coupled Model Intercomparison Project (CMIP)](https://www.wcrp-climate.org/wgcm-cmip),
+the tool can also be used to analyze output from regional models, a large variety of gridded observational data
+and reanalysis datasets. The development of ESMValTool is supported by several
+[national and international projects](https://www.esmvaltool.org/acknowledgements).
+ESMValTool has been in several chapters of the
+[Assessment Report 6 of the IPCC’s WG1](https://www.ipcc.ch/report/sixth-assessment-report-working-group-i/).
 
 ## Main Features
```

Comment:
We have removed any reference to CMIP in the new bullet list. Is that ok?

Reply:
Yes, that was on purpose. I figured it might be enough to mention CMIP only in the description above but not giving it too much weight by also having them mentioned in the bullet list.